util/build_tock.sh

```diff
@@ -24,4 +24,5 @@ cargo +"$(cat ${TOOLCHAIN_FILE})" build \
 	--target "${TARGET}" \
 	--manifest-path "${MANIFEST_PATH}" \
 	--target-dir "${TARGET_DIR}" \
+  --features="${TOCK_FEATURES}" \
```

Comment:
Nit: indentation, presumably this file uses tabs, though I'm not sure why.

Reply:
Done.